tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -93,7 +93,7 @@ public function testGeneratesForeignKeySqlForNonStandardOptions(): void
         );
         self::assertEquals(
             'CONSTRAINT my_fk FOREIGN KEY (foreign_id)'
-            . ' REFERENCES my_table (id) DEFERRABLE INITIALLY IMMEDIATE',
+            . ' REFERENCES my_table (id) DEFERRABLE',
```

Comment:
`DEFERRABLE` implies `INITIALLY IMMEDIATE`, so not generating it is backward-compatible.